src/index.js

```diff
@@ -165,6 +165,9 @@ function generate(dataFrom, config = defaultConfig) {
   let definitions = moduleDefs.map(def => def.code).join('\n').replace(/\n(\s+)\n/g, '\n');
 
   if(postprocessing) definitions = postprocessing(definitions, dataFrom);
+
+	if(collection.filter( issue => issue.type === 'error').length) throw new Error('YOU HAVE FAILED THIS CITY')
```

Comment:
Why not throw the collection?  So we can see what's wrong.  The stack trace which print out tells nothing.  Maybe even throw a custom error which can be caught and the erorrs logged there, instead of as they occur.